### ISSUE TITLE
Where does the training data come from?

### ISSUE THREAD

**Issue by polm** [Hironsan/asari#6]
What does the model here use as training data? Didn't see an explanation anywhere...

**Comment by othmane-ab** [comment]
It appears he didn't add the training data in the repository

**Comment by HeroadZ** [comment]
hi, there. @polm  See you again.
I'm a beginner of NLP and interested in japanese. But I can't find dataset with labels in japanese for sentiment analysis.
Could you give me some advices? 😄

**Comment by polm** [comment]
Sorry, I am not aware of any good training set, which is why I was curious what was used for this model. [pymlask](https://github.com/ikegami-yukino/pymlask) is rule based but seems solid. 

Since you are affiliated with an academic institution you may have access to this Rakuten dataset. 

https://rit.rakuten.co.jp/data_release_ja/

**Comment by HeroadZ** [comment]
> Sorry, I am not aware of any good training set, which is why I was curious what was used for this model. [pymlask](https://github.com/ikegami-yukino/pymlask) is rule based but seems solid.

I just saw an article which compare the current japanese sentiment analysis tools. It seems like that there exist problems for the pymlask. If test sentence doesn't contain any word in dictionary, the polarity is None. If you are interested, check [this](https://qiita.com/hnishi/items/0d32a778e375a99aff13).

@ikegami-yukino how do you think about the article above? Thank you for your work.

> Since you are affiliated with an academic institution you may have access to this Rakuten dataset. https://rit.rakuten.co.jp/data_release_ja/

Thank you for your advices. But maybe I cannot access it because the research I'm doing now is from a company where I work as an intern. 😢

**Comment by polm** [comment]
If you want to close this that's fine, but I still don't see an explanation of the training data anywhere. Did I miss it?
